wum-sce-test-wso2is-5.7.0-full-testgrid.yaml

```diff
@@ -28,6 +28,13 @@ infrastructureConfig:
             KeyPairName: "deployment-prod"
             CertificateName: "wso2cert"
             DBUsername: "wso2carbon"
+        - name: 'prod-wso2is-samples-deployment'
+          description: ''
+          type: CLOUDFORMATION
+          file: product-scenarios/intg-test-resources/test-is-samples.yml
```

Comment:
This location does not exist under this repo [1] as I see. Shall we keep all deployment scripts under one repo?

[1] https://github.com/wso2-incubator/testgrid-job-configs/pull/37/files#diff-b8acf535f18a39c01bc50b246541d10fR17

Reply:
Added the script in https://github.com/wso2/aws-is/pull/17